### ISSUE TITLE
Release Notes for v2.6.0, YAML config and debug mode 

### ISSUE THREAD

**Issue by williamfgc** [ornladios/ADIOS2#2300]
@chuckatkins @pnorbert thanks for the release. I'll work on the packaging tasks.
Just wanted to make sure that an entry for YAML config support is added as it's currently not present in the docs.
Also, I'm planning to exploit debug mode for our use cases, please don't completely remove it in future releases. A simple benchmark in  #2127 shows some impact as the number of variables increase, profiling and doing a PR has been in my wishlist for a while. It would be good to share the benchmarks that drove that decision (stated in the release notes) as I could add more cases in the future. Thanks!

**Comment by chuckatkins** [comment]
> Just wanted to make sure that an entry for YAML config support is added as it's currently not present in the docs.

Even though the release is already published, I thought you can still edit the release notes since you're permission level is `maintainer`.

**Comment by williamfgc** [comment]
@chuckatkins thanks, I just wanted to check with you first.

**Comment by chuckatkins** [comment]
Although I believe the yaml config was present in 2.5.0 wasn't it?

**Comment by williamfgc** [comment]
@chuckatkins it was added not long after to master. This is the first release.

**Comment by chuckatkins** [comment]
Closing this as the release notes have been updated for the YAML information; thanks for updating them.  W.r.t debug mode, it was already decided in other issues, PRs, and developer meetings to remove from the constructor API and the existing implementation (outside of scope here as this issue is specifically about the release notes), which is what the release notes reflect.

However, we did leave open the prospect of having more expensive runtime checks in the future that could be enabled or disabled through other means (environment variable, config file parameter, etc.), just not through the constructor API.
